### PR TITLE
Changed the directory structure in noobaa storage plugin 

### PR DESCRIPTION
**Pull request by afreen23**
```
└── src
    ├── components
    │   ├── dashboard-page
    │   │   └── object-service-dashboard
    │   │       └── ...
    │   ├── object-bucket-claim-page
    │   │   ├── ...
    │   └── object-bucket-page
    │       ├── ....
    ├── constants
    │   ├── constants.ts
    │   └── table-filters.ts
    ├── models.ts
    ├── plugin.ts
    ├── queries.ts
    ├── selectors
    │   ├── buckets-pages.ts
    │   ├── dashboard.ts
    │   └── index.ts
    └── utils
        └── bucket-pages.ts
```

### CODE REVIEW

**Comment by spadgett** [comment]
@afreen23 can you pull the bug fix out to a separate PR? the refactor should wait until we have a branch for 4.3

**Comment by joshuawilson** [comment]
Whenever you refactor (move or delete) files when using git to store the source code, you should always track the move or delete in it's own commit. Then any change that you need to make in a new one. Moving stops the git history and so you want to track your changes after the move is committed.

**Comment by afreen23** [comment]
@spadgett please remove the hold now.
Thanks!

File: frontend/packages/noobaa-storage-plugin/src/selectors/dashboard.ts [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import * as _ from 'lodash';
 import { Alert } from '@console/internal/components/monitoring';
 import { PrometheusResponse } from '@console/internal/components/graphs';
-import { K8sResourceKind } from '@console/internal/module/k8s';
 
 export const filterNooBaaAlerts = (alerts: Alert[]): Alert[] =>
```
Comment (cloudbehl) [inline frontend/packages/noobaa-storage-plugin/src/selectors/dashboard.ts]: This can also move into utils as this is not a selector

Comment (afreen23) [reply frontend/packages/noobaa-storage-plugin/src/selectors/dashboard.ts]: right! good one :)

**Pull request closed without merging**